src/js/odata.js

```diff
@@ -1597,22 +1597,22 @@ odata.prototype.errorHandler = function(evt) {
  * The purpose of this method is to perform file validation
  */
 odata.prototype.validateFileName = function(fileName, fileSize) {
-    var letters = /^[一-龠ぁ-ゔ[ァ-ヴー々〆〤0-9a-zA-Z-_.]+$/;
+    var letters = new RegExp("^[^" + objCommon.getValidateBlackList() + "]+$");
     //var letters = /^[0-9a-zA-Z-_.]+$/;
     if (fileName.indexOf("fakepath") != -1) {
         fileName = fileName.substring(fileName.lastIndexOf("fakepath") + 9);
     }
     var fileLength = fileName.length;
     objOdata.fileType = fileName.substring(fileName.lastIndexOf('.') + 1);
-    if (fileLength > 128) {
+    if (fileLength > 256) {
         objOdata.spinner.stop();
         document.getElementById("dvGreyOut").style.display = 'none';
         alert("File name can not exceed 128 characters");
```

Comment:
Please change message to the following.
"File name cannot exceed 256 characters"